coding-concepts.js

```diff
@@ -8,48 +8,69 @@
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: 10 : )))) it will return the leangth of the string in amount of index spaces it the string is taking up.
+// b) Verify and explain: .length just asks the length i do not remember having quastions about this command, so it is a pritty easy command to see what it does.
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "o"!? For a sure thing it would return the value of the sintring's index, but than what do i know. 
+// b) Verify and explain: and yes icheese a correct answer yeay!!! However i was a little skeptical because in class we found out that in the "fruit loop" aka for loops that i-returns index and [i]-returns location (this made sense in my head and in my notes, now it actualy is again not making any sense, need to go back and look at the lecture)***
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+console.log(languages[index])
+// console.log(languages(index))
 
-// a) Your answer:
-// b) Verify and explain:
+
+// a) Your answer: "Ruby"?!!! Well it better return ruby, because by all none making sense making in my head it is starting to click. so if i returns the index and the [i] returns the location, maybe it returns the location value?
+// b) Verify and explain: well, sticks and stones will have to break something because console.log(languages(index)) is returning : TypeError: languages is not a function... hmmmmmmmmmmmm .... double the hmmmm time. 
+
+const returnMyLanguageBackToMeAndPleseMakeSense = () => {
+    return(languages[index])
+}
+console.log(returnMyLanguageBackToMeAndPleseMakeSense(index))
+
+//i think i just want to confuse myself evenmore. why [] and () return the same thing in defferent situations and returning error in others? is just syntax that i need to remember and chill?! yes... i am going to get some coffe and relax
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
+var weekendDays = ["saturday", "sunday"]
+// this aint working
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// i wanna try something: nevermind
+// console.log(weekendDays.toUpperCase("")) 
+
+// a) Your answer: ["SATURDAY", "SUNDAY"] .toUpperCase is a built in method that needs none of my  overthinking skills, so i will just leave it at that.
+// b) Verify and explain: is this a trick quastion? i am getting an error: weekendDays.toUpperCase is not a function.? WIGO (aka. what is going on?). OVERTHINKING SKILLS AKTIVATE!!!!!
```

Comment:
Kind of a trick question, haha. `.toUpperCase()` can only be used on strings, so here when it tries to use it on an array, it returns the error.